hat/conf/v26.routes

```diff
@@ -79,5 +79,6 @@ GET           /she/static/$endpoint<[0-9a-z-/]+>
 
 
 # Contract Data
-POST          /contract-data/read/$namespace<[0-9a-z-]+>/$endpoint<[0-9a-z-/]+>              org.hatdex.hat.api.controllers.RichData.getContractData(namespace, endpoint, orderBy: Option[String], ordering: Option[String], skip: Option[Int], take: Option[Int])
-POST          /contract-data/create/$namespace<[0-9a-z-]+>/$endpoint<[0-9a-z-/]+>            org.hatdex.hat.api.controllers.RichData.saveContractData(namespace, endpoint, skipErrors: Option[Boolean])
+POST          /contract-data/read/$namespace<[0-9a-z-]+>/$endpoint<[0-9a-z-/]+>              org.hatdex.hat.api.controllers.ContractData.readContractData(namespace, endpoint, orderBy: Option[String], ordering: Option[String], skip: Option[Int], take: Option[Int])
+POST          /contract-data/create/$namespace<[0-9a-z-]+>/$endpoint<[0-9a-z-/]+>            org.hatdex.hat.api.controllers.ContractData.createContractData(namespace, endpoint, skipErrors: Option[Boolean])
+PUT           /contract-data/update/$namespace<[0-9a-z-]+>                                   org.hatdex.hat.api.controllers.ContractData.updateContractData(namespace)
```

Comment:
Non Blocking: Im unsure about this API. On one hand it is consistent but on the other is not RESTFUL. My understanding is that a RESTFUL API would not have function names (/update, /create, /read) like a SOAP API would but instead use the HTTP verbs. So you could have POST call create and PUT call update without the need to specify them in the URL. The issue is that the API should also be consistent and we cannot remove /create and /read as that would break backwards compatability, so I can also see an arguement for leaving it as is. Also its missing and EOL unsure if that is a big deal or not

Reply:
I have no argument against the REST designs and you pointing them out.

The `recordId(s)` in the body of the PUT correlate to the data that is getting updating.  So we are pretty far from the REST design here.  

What I did do was attempt to be obvious with what we have already.

To move to something more RESTful, I will open a ticket for redesigning the API to be such.

Reply:
DAS-177 created.